doc/source/conf.py

```diff
@@ -129,8 +129,8 @@
 
 # configure pymechanical for embedding
 ansys.mechanical.core.BUILDING_GALLERY = True
-app = ansys.mechanical.core.App(version=232)
 if "PYMECHANICAL_BUILDING_GALLERY_GITHUB" in os.environ:
```

Comment:
just remove this whole block, we found that PYMECHANICAL_BUILDING_GALLERY_GITHUB is never set anywhere